src/lib/rooibos/TestGroup.ts

```diff
@@ -1,49 +1,46 @@
-import type { AstEditor, CallExpression, DottedGetExpression, Expression, NamespaceContainer, Scope } from 'brighterscript';
-import { ArrayLiteralExpression, createInvalidLiteral, createStringLiteral, createToken, isDottedGetExpression, TokenKind, isFunctionExpression, Parser, ParseMode, util } from 'brighterscript';
-import * as brighterscript from 'brighterscript';
-import { BrsTranspileState } from 'brighterscript/dist/parser/BrsTranspileState';
+import type { AstEditor, CallExpression, DottedGetExpression, Expression, FunctionExpression, NamespaceContainer, Scope } from 'brighterscript';
+import { ArrayLiteralExpression, createInvalidLiteral, createStringLiteral, createToken, isDottedGetExpression, TokenKind, isFunctionExpression, Parser, ParseMode, util, createVisitor, isCallExpression, WalkMode, createVariableExpression, isCallfuncExpression, isVariableExpression } from 'brighterscript';
 import { diagnosticErrorProcessingFile } from '../utils/Diagnostics';
 import type { RooibosAnnotation } from './Annotation';
 import type { TestCase } from './TestCase';
-import type { TestSuite } from './TestSuite';
 import { TestBlock } from './TestSuite';
+import type { TestSuite } from './TestSuite';
 import { getAllDottedGetParts, getRootObjectFromDottedGet, getStringPathFromDottedGet, sanitizeBsJsonString } from './Utils';
 
 export class TestGroup extends TestBlock {
 
     constructor(testSuite: TestSuite, annotation: RooibosAnnotation) {
         super(annotation);
         this.testSuite = testSuite;
-        this.setupFunctionName = this.setupFunctionName || this.testSuite.setupFunctionName;
-        this.tearDownFunctionName = this.tearDownFunctionName || this.testSuite.tearDownFunctionName;
-        this.beforeEachFunctionName = this.beforeEachFunctionName || this.testSuite.beforeEachFunctionName;
-        this.afterEachFunctionName = this.afterEachFunctionName || this.testSuite.afterEachFunctionName;
+        this.setupFunctionName ||= this.testSuite.setupFunctionName;
+        this.tearDownFunctionName ||= this.testSuite.tearDownFunctionName;
+        this.beforeEachFunctionName ||= this.testSuite.beforeEachFunctionName;
+        this.afterEachFunctionName ||= this.testSuite.afterEachFunctionName;
```

Comment:
These should actually be `??= instead of `||=` since we are setting setupFunctionName to a value, not boolean. (yes, `||` gets us similar, but it's not as clear semantically).